log4j-core/src/main/java/org/apache/logging/log4j/core/script/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Log4j 2 Script support.
  */
 @Export
-@Version("2.20.2")
+@Version("2.25.0")
```

Comment:
```suggestion
@Version("2.26.0")
```
Tonight I'll release `2.25.0`, so the next minor version will become `2.26.0`.